tests/e2e/gauntlet_deployer.go

```diff
@@ -41,7 +41,8 @@ type InspectionResult struct {
 }
 
 // GetDefaultGauntletConfig gets the default config gauntlet will need to start making commands
-// 	against the environment
+//
+//	against the environment
```

Comment:
Is this an incidental reformat due to the extra leading spaces?
```suggestion
// against the environment
```
Same below, although that one still makes sense :shrug:

Reply:
I ran it through the `gopls` formatter and that's what I got 🤷🏻